lib/Runtime/Language/AsmJsLink.cpp

```diff
@@ -143,18 +146,21 @@ namespace Js{
         }
 
 #ifdef ENABLE_SIMDJS
-        Var asmSimdObject = JavascriptOperators::OP_GetProperty(stdlib, PropertyIds::SIMD, scriptContext);
```

Comment:
So if I understand correctly, the issue is here where we get the SIMD even if none are used right ?